src/Hook/ActionAdminLoginControllerSetMedia.php

```diff
@@ -139,4 +147,40 @@ protected function trackEditionLoginPage($local = false)
             }
         }
     }
+
+    /**
+     * Check if admin URL segment changed and update if needed
+     *
+     * @return void
+     */
+    protected function checkAndUpdateUrlIfNeeded()
+    {
+        try {
+            /** @var \PrestaShop\Module\PsAccounts\Account\StatusManager $statusManager */
+            $statusManager = $this->module->getService(\PrestaShop\Module\PsAccounts\Account\StatusManager::class);
+            /** @var \PrestaShop\Module\PsAccounts\Provider\ShopProvider $shopProvider */
+            $shopProvider = $this->module->getService(\PrestaShop\Module\PsAccounts\Provider\ShopProvider::class);
+
+            $status = $statusManager->getStatus(false, \PrestaShop\Module\PsAccounts\Account\StatusManager::CACHE_TTL, 'ps_accounts');
+
+            // Only check if shop is already verified
+            if (!$status->isVerified) {
+                return;
+            }
+
+            $shopId = \Shop::getContextShopID();
+            if (ShopUrl::urlChanged($status, $shopProvider->getUrl($shopId))) {
```

Comment:
Default value ignore backoffice URL, too bad 💩 
```suggestion
            if (ShopUrl::urlChanged($status, $shopProvider->getUrl($shopId), true)) {
```
And shouldn't we check ONLY backoffice URL here ?